middleware/database/migrations/2023_10_13_202359_create_provider_content_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateProviderContentTables extends Migration
+{
+    public function up()
+    {
+        Schema::create('provider_content', function (Blueprint $table) {
+            $table->id();
+            $table->integer('content_id');
```

Comment:
Should these be strings to support both integers id's and string uuids from external ids?

Reply:
content_id and external_resource_id